helm/charts/hydra/values.yaml

```
@@ -291,8 +291,12 @@ deployment:
     enabled: false
     minReplicas: 1
     maxReplicas: 3
-  # targetCPUUtilizationPercentage: 80
-  # targetMemoryUtilizationPercentage: 80
+    targetCPU:
+      type: Utilization
+      averageUtilization: 80
+    targetMemory:
+      type: Utilization
+      averageUtilization: 80
```

I like the format :) However, as with resources i don't think we should set any default values, and leave it for the users to provide them. I know it is easier to provide "sane" default values for hpa then for resources, but nevertheless imho we should leave that to the user to decide. 

```suggestion
    targetCPU: {}
      # type: Utilization
      # averageUtilization: 80
    targetMemory: {}
      # type: Utilization
      # averageUtilization: 80
```

Could you update this for other values? And in the hacks, there we can provide some values as parameters :)